test/unit/config.spec.js

```diff
@@ -20,6 +20,30 @@ describe("config", function () {
     cloudinary.config(true);
   });
 
+  it("should support `hide_sensitive` config param", function () {
+    const config = cloudinary.config({hide_sensitive: true});
+    expect(config.hide_sensitive).to.eql(true)
+  });
+
+  it("should hide API key and secret upon error when `hide_sensitive` is true", async function () {
+    try {
+      cloudinary.config({hide_sensitive: true});
+      const result = await cloudinary.v2.api.resource("?");
+      expect(result).fail();
+    } catch (err) {
+      expect(err.request_options).not.to.have.property("auth");
+    }
+  });
+
+  it("should hide Authorization header upon error when `hide_sensitive` is true", async function () {
+    try {
+      cloudinary.config({hide_sensitive: true});
+      const result = await cloudinary.v2.api.resource("?", { oauth_token: 'MTQ0NjJkZmQ5OTM2NDE1ZTZjNGZmZjI4' });
```

Comment:
Actual value of the oath token is irrelevant, so we can set it to `irrelevant`.

Also, be extra careful about committing secrets to a public repo. They'll be here forever ;)

Reply:
If it was used before, we can extract it as a variable to keep it in a single place and name it accordingly to make sure that everyone knows we're not leaking secrets there.

That being said, if the value of the token is irrelevant for the test, I'd keep `irrelevant`. More explicit.